core/src/main/java/io/grpc/ProxyDetector.java

```diff
@@ -14,8 +14,9 @@
  * limitations under the License.
  */
 
-package io.grpc.internal;
+package io.grpc;
 
+import io.grpc.internal.ProxyParameters;
```

Comment:
ProxyParameters would need to to be promoted to io.grpc, since io.grpc cannot depend on io.grpc.internal.